pkg/v1/tkg/tkgctl/init.go

```diff
@@ -69,7 +69,7 @@ func (t *tkgctl) Init(options InitRegionOptions) error {
 
 	ceipOptIn, err := strconv.ParseBool(options.CeipOptIn)
 	if err != nil {
-		ceipOptIn = false
+		ceipOptIn = true
```

Comment:
What if I am a TCE user and I typed the CEIPOptIn flag value incorrectly. Will it set the value to `true` by default?

Reply:
Good catch. I've added a condition for this